src/main/java/com/example/git_repo/retryer/GitRetryer.java

```diff
@@ -0,0 +1,12 @@
+package com.example.git_repo.retryer;
+
+import feign.Retryer;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class GitRetryer extends Retryer.Default {
```

Comment:
Zastanow się czy jest to ci potrzebne tak naprawdę na ten moment nic tutaj nie zmieniasz wiec nie musisz tworzyc takiej klasy mogles od razu w definicji beana wskazać return new Retryer.Default(100, SECONDS.toMillis(1), 5)